x-pack/test/functional/apps/discover/reporting.ts

```diff
@@ -103,8 +103,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.discover.selectIndexPattern('ecommerce');
       });
 
-      // Discover defaults to short urls - is this test helpful? Clarify in separate PR
-      xit('generates a report with single timefilter', async () => {
+      it('generates a report with single timefilter', async () => {
```

Comment:
It looks like the serverless version of this test in `x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts` hasn't been unskipped yet.

Also there were some other minor changes to these files in #180406, like removing `gte:now-24h/h,lte:now))))` from the `reportURL` check, and removing `await PageObjects.reporting.checkForReportingToasts()` between tests. Can any of these be restored, or is the behaviour different now?

Reply:
Looks like this question has been partially resolved: the serverless test isn't skipped.

I also wonder about the minor change, and wonder if we can restore the validation of the sharing URL that @davismcphee wrote about:
```diff
diff --git x-pack/test/functional/apps/discover/reporting.ts x-pack/test/functional/apps/discover/reporting.ts
index 0b193e5b850..ab5ee99448b 100644
--- x-pack/test/functional/apps/discover/reporting.ts
+++ x-pack/test/functional/apps/discover/reporting.ts
@@ -130,7 +130,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(timeFiltersNumberInReportURL).to.be(1);
         expect(
           decodeURIComponent(reportURL).includes(
-            'query:(range:(order_date:(format:strict_date_optional_time'
+            'query:(range:(order_date:(format:strict_date_optional_time,gte:now-24h/h,lte:now))))'
           )
         ).to.be(true);
 
diff --git x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts
index c0e99978750..1a500817ddd 100644
--- x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts
+++ x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts
@@ -143,7 +143,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(timeFiltersNumberInReportURL).to.be(1);
         expect(
           decodeURIComponent(reportURL).includes(
-            'query:(range:(order_date:(format:strict_date_optional_time'
+            'query:(range:(order_date:(format:strict_date_optional_time,gte:now-24h/h,lte:now))))'
           )
         ).to.be(true);
 
```

Reply:
Thank you for pointing this out @davismcphee! Turns out you found a potential issue and now is fixed in [510d054](https://github.com/elastic/kibana/pull/183643/commits/510d054ff919d81cad1112592254692de941bfd4). The tests are restored to show the absolute time of now etc. as it should be for watcher. Thanks!!

Reply:
I synced up with @rshen and we figured out we can restore the `reportURL` expectation by fixing a bug in `packages/kbn-reporting/public/share/share_context_menu/register_csv_modal_reporting.tsx`, line 145:

```diff
diff --git packages/kbn-reporting/public/share/share_context_menu/register_csv_modal_reporting.tsx packages/kbn-reporting/public/share/share_context_menu/register_csv_modal_reporting.tsx
index 6877ed1c2b3..f0939204d83 100644
--- packages/kbn-reporting/public/share/share_context_menu/register_csv_modal_reporting.tsx
+++ packages/kbn-reporting/public/share/share_context_menu/register_csv_modal_reporting.tsx
@@ -140,9 +140,10 @@ export const reportingCsvShareProvider = ({
 
       const reportingUrl = new URL(window.location.origin);
 
       const relativePath = apiClient.getReportingPublicJobPath(
         reportType,
-        apiClient.getDecoratedJobParams(getJobParams())
+        apiClient.getDecoratedJobParams(getJobParams(true))
       );
 
       const absoluteUrl = new URL(relativePath, window.location.href).toString();
```